File: insuranceclaim.client/src/app/Models/PaymentModel.ts
export interface IPaymentRequestModel {
  id: number,
  paymentDamage: number,
  paymentClaimantCost: number,
  paymentDefenceCost: number,
  paymentType: string
}


export interface IPaymentModel {
  id: number,
  paymentDamage: number,
  paymentClaimantCost: number,
  paymentDefenceCost: number,
  isInApproval: boolean,
  status: string,
  statusDate: Date,
  paymentType: string

}

File: InsuranceClaim.Server/Models/PaymentModel.cs
namespace InsuranceClaim.Server.Models
{
    public class PaymentModel
    {
        public int Id { get; set; }
        public double PaymentDamage { get; set; }
        public double PaymentClaimantCost { get; set; }
        public double PaymentDefenceCost { get; set; }
        public string Status { get; set; }

        public Boolean IsInApproval { get; set; }

        public string PaymentType { get; set; }

        public DateTime StatusDate { get; set; }

        //public PaymentModel() { }
    }

    public class AddPaymentRequest
    {
        public int Id { get; set; }
        public double PaymentDamage { get; set; }
        public double PaymentClaimantCost { get; set; }
        public double PaymentDefenceCost { get; set; }
        public string PaymentType { get; set; }
    }
}
